src/Core/Framework/App/Command/ChangeShopIdCommand.php

```diff
@@ -13,30 +13,35 @@
 use Symfony\Component\Console\Output\OutputInterface;
 
 /**
- * @internal only for use by the app-system
+ * @internal
+ *
+ * @deprecated tag:v6.8.0 - The command's name will change to `app:shop-id:change`
  */
 #[AsCommand(
     name: 'app:url-change:resolve',
```

Comment:
maybe we should rename the command already, and add the old name as alias. then the alias can simply be removed with 6.8